docs/source/guides/userguide.rst

```diff
@@ -20,8 +21,73 @@ then a :obj:`~pyDeltaRCM.deltaRCM_driver.pyDeltaRCM` model instance initialized
 Multiple parameters can be specified line by line.
 
 Default values are substituted for any parameter not explicitly given in the ``input_file`` ``.yaml`` file.
-Default values are defined as:
+Default values of the YAML configuration are listed in the :doc:`../reference/deltaRCM_driver/yaml_defaults`.
 
-.. literalinclude:: ../../../pyDeltaRCM/default.yml
-   :language: yaml
-   :linenos: 
+
+===================
+Starting model runs
+===================
+
+There are three ways to interact with the pyDeltaRCM model. 
+Two methods are part of the "high-level" model API, and these approaches each consist of a single line of code.
+The third method is to create a model instance via the "low-level" model API and manually handle timestepping and model finalization.
+
+
+High-level model API
+====================
+
+The high-level API is accessed via either a shell prompt or python script, and is invoked directly if a YAML configuration file includes the ``timesteps`` variable.
+
+.. note::
+    To use the high-level API, the configuration YAML file must contain the ``timesteps`` variable.
+
+Bash API
+--------
+
+ For example, to invoke a model run from the bash prompt using the YAML file ``model_configuration.yml`` which looks like:
+
+.. code-block:: yaml
+
+    timesteps: 500
+
+we would simply invoke at the bash prompt:
+
+.. code:: bash
+    
+    python -m pyDeltaRCM --config model_configuration.yml
+
+This invokation will run the pyDeltaRCM with the parameters specified in the ``model_configuration.yml`` file, and automatically :obj:`~pyDeltaRCM.deltaRCM_driver.update` the model 500 times.
```

Comment:
Checked this out and built the docs locally - for me the :obj: did not end up being a link in the compiled HTML (true of this object and the ones below in the Python API section). Not an issue if it works fine for you (my Ruby gem might be out of date)

Reply:
thanjs, this is because the object (method) I was trying to find is `pyDeltaRCM.deltaRCM_driver.pyDeltaRCM.update` and not `pyDeltaRCM.deltaRCM_driver.update`. I have fixed it.